src/main/java/uk/ac/ebi/eva/pipeline/ParameterFromProperties.java

```diff
@@ -0,0 +1,68 @@
+package uk.ac.ebi.eva.pipeline;
```

Comment:
Shall we keep this class in the root package or move it to `uk.ac.ebi.eva.pipeline.parameters`?